jwst/datamodels/multiprod.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import absolute_import, unicode_literals, division, print_function
+
+from . import model_base
+from .image import ImageModel
+
+
+__all__ = ['MultiSlitModel']
+
+
+class MultiProductModel(model_base.DataModel):
+    """
+    A data model for multi-DrizProduct images.
+
+    This model has a special member `products` that can be used to
+    deal with each DrizProduct at a time.  It behaves like a list::
+
+       >>> multiprod_model.products.append(image_model)
+       >>> multislit_model.products[0]
+       <DrizProductModel>
+
+    If `init` is a file name or an `DrizProductModel` instance, an empty
+    `DrizProductModel` will be created and assigned to attribute `products[0]`,
+    and the `data`, `wht`, `con`, and `relsens` attributes from the
+    input file or `DrizProductModel` will be copied to the first element of
+    `products`.
+
+    Parameters
+    ----------
+    init : any
+        Any of the initializers supported by `~jwst.datamodels.DataModel`.
+    """
+    schema_url = "multiproduct.schema.yaml"
+
+    def __init__(self, init=None, **kwargs):
+        if isinstance(init, ImageModel):
```

Comment:
Shouldn't this be "DrizProductModel"? An ImageModel doesn't have the WHT and CON attributes that are referenced later.